src/forge/cargo.rs

```diff
@@ -54,7 +55,12 @@ impl Forge for CargoForge {
         let settings = Settings::get();
         settings.ensure_experimental("cargo backend")?;
         let cmd = if self.is_binstall_enabled() {
-            CmdLineRunner::new("cargo-binstall").arg("-y")
+            let runner = CmdLineRunner::new("cargo-binstall").arg("-y");
+            if let Some(token) = &*GITHUB_TOKEN {
+                runner.env("GITHUB_TOKEN", token)
```

Comment:
```suggestion
                runner = runner.env("GITHUB_TOKEN", token)
```

I prefer making it mutable and doing it this way, makes later edits simpler